geode-pulse/src/test/java/org/apache/geode/tools/pulse/internal/data/MemberTest.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.tools.pulse.internal.data;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+
+public class MemberTest {
+  private Cluster.Member member;
```

Comment:
Cluster.Member and Cluster.Client look like good candidates for moving to top-level classes. They don't seem be appropriate as inner classes.

Reply:
@kirklund : agreed but, as I said in https://github.com/apache/geode/pull/3694#issuecomment-500628139, I don't see any value in adding those changes within this particular pull request. The `Cluster` class has several inner classes that should be moved to top-level if you ask me, but it's a refactor that (from my perspective at least) should be done through another JIRA if you ask me.